pkg/engine/internal/planner/planner_test.go

```diff
@@ -156,8 +156,10 @@ TopK sort_by=builtin.timestamp ascending=false nulls_first=false k=1000
     └── TopK sort_by=builtin.timestamp ascending=false nulls_first=false k=1000
         └── Compat src=metadata dst=metadata collision=label
             └── ScanSet num_targets=2 predicate[0]=GTE(builtin.timestamp, 2025-01-01T00:00:00Z) predicate[1]=LT(builtin.timestamp, 2025-01-01T01:00:00Z)
-                    ├── @target type=ScanTypeDataObject location=objects/00/0000000000.dataobj streams=5 section_id=1 projections=()
-                    └── @target type=ScanTypeDataObject location=objects/00/0000000000.dataobj streams=5 section_id=0 projections=()
+                ├── DataObjScan location=objects/00/0000000000.dataobj streams=5 section_id=1 projections=()
```

Comment:
targets are now children, not comments

Reply:
I'm a little worried about that; they're not children nodes and it might give someone an incorrect understanding of the plan DAG. 

I saw your comment above about moving them to children since we're using comments for other things, but I feel like comments can still make sense here, especially since they're called out explicitly as `@target`. WDYT?